styles/style.css

```diff
@@ -0,0 +1,246 @@
+* {
+    margin:0;
+    padding: 0;
+}
```

Comment:
consider also adding a `box-sizing: border-box;` rule to this as well. This can help us reason more easily about the sizes that content areas occupy. This isn't the default because the DOM originally used the `content-box` model, in which the width and height applied only to the content of a tag, and the border and padding were handled separately. It was always confusing, but since it was the default, we need to explicitly set our elements to use the more modern `border-box`;